weasyl/test/test_collection.py

```diff
@@ -19,7 +19,22 @@ def offer(self):
         collection.offer(self.creator, self.s, self.collector)
 
     def count_collections(self, pending, rating=ratings.GENERAL.code):
-        return len(collection.select_manage(self.collector, rating, 10, pending))
+        return self.db.scalar(
```

Comment:
This feels a little brittle. Is there nothing in the collection module that's suitable?